src/pages/Register/index.tsx

```diff
@@ -40,11 +41,17 @@ const Register = () => {
     }
 
     if (role === jobseeker) {
-      dispatch(registerJobseekerRequest(email, password))
-      setMessage('Registered successfully')
+      if (password === confirmPassword) {
```

Comment:
Instead of repeating this line again below, you can check for `confirmPassword` first, then check the role

Reply:
yeah, thats good, i will